framework/files/etc/luet/luet.yaml

```diff
@@ -0,0 +1,16 @@
+repositories:
+  - &cos
+    name: "cos"
+    description: "cOS official"
```

Comment:
Shouldn't be `Elemental` instead of `cOS`?

Reply:
Mmh, this are generic repos based off the teal flavor so they are generic enough that they can be used by any derivativate that has a sle micro base. So not really elemental related but maybe a better name could be used like upstream repos or elemental-toolkit repos